docs/source/installing_more.rst

```diff
@@ -1,8 +1,8 @@
-.. _installing_more:
+:orphan:
```

Comment:
what does :orphan: does? better than having nothing?

Reply:
:orphan: suppresses the "document is not part of any toctree" in sphinx. This is the only solution I found to that problem.